yext/answers
# FilterOptions: filter by locationRadius w/ optionType RADIUS_FILTER

This commit adds a new config option for FilterOptions called optionType, which
can be either RADIUS_FILTER or STATIC_FILTER, defaulting to STATIC_FILTER which
is the original behavior. Filtering by radius is done by appending a new
locationRadius param to a query.

TEST=manual
Test Facets, FilterBox, FilterOptions
Facets: smoke test
FilterBox:
- check that regular STATIC_FILTER FilterBox still works, and can filter by distance
- check that reseting FilterBox then applying also clears the locationRadius
- check that works with searchOnChange (after fixing reset button init bug with #768 )

FilterOptions:
- check that defaults to STATIC_FILTER, throws helpful errors
when trying to use unrecognized optionType or trying to use multioption
with RADIUS_FILTER
- check that setting a value of 0 for RADIUS_FILTER clears locationRadius
- check that selected filter persists over page refresh
- check that can filter by locationRadius by searching "new york" under vertical key "people"
  locationRadius = 1 gives no results, locationRadius = 8046.72 = 1 result
- check that clicking the reset button in FilterOptions then searching again clears the 
  locationRadius
